procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+
+[decisions]: http://example.com (**TODO**: link when decision making process is documented)
+
+### Meeting structure
+
+In addition to the main agenda items, every meeting has the following structure:
+
+- **Check-in**
+  - The check-in is a chance for Council members to connect on a human level and share how their doing outside of the context of the meeting. Examples of things shared in the check-in include: "I'm not feeling well because of my allergies, and so I'm not in a good headspace", "My partner and I just adopted a cat so I'm doing great!", and "I'm doing fine - nothing to report".
+- **Assigning roles**
+  - The Council members decide by consensus who will fill the roles of facilitator and note taker.
+- **Turn on recording**
+  - Recording is started
+- **Consent to the agenda**
+  - This is a chance for last minute changes to the agenda to be made through consensus and to ensure everyone is on the same page as far as what will be discussed. Items may be added or struck from the agenda through consensus though it is advised that additions to the agenda be small in scope (to ensure that everyone involved has had enough time to prepare for whatever will be discussed) and that additions only be accepted if consent can be reached quickly (to ensure the meeting starts in a timely manner).
+- **Main agenda**
+  - The main topics of the meeting which have just been consented to
+- **Clarifications**
+  - A call is made for any last minute clarifications, questions, requests for additional action.
+- **Turn off recording**
+  - The recording is stopped
+- **Check-out**
+  - This is a chance for council members to express how they felt the meeting went including areas for improvement, particular highlights and kudos, or any other relevant meta aspects of the meeting.
+
+A template for this meeting structure that can be used to boostrap the agenda making process can be found [here](../templates/sync-meeting-agenda.md).
+
+## Meeting Decorum 
+
+The following are the social expectations that are expected of meeting participants:
+
+- **Usage of "hands"**: all participants are expected to use the "raise hands" feature of the video meeting software to put themselves into a queue to be called on by the facilitator. There are a few exceptions to this however: the facilitator when performing the duties of that role and the scribe when asking clarifying questions.
+- **Points of order**: every participant is encouraged to raise "points of order"  which are meta commentary on some aspect of the meeting that needs clarification or discussion. If a point of order is considered urgent, it may be called out verbally - bypassing the usual queuing mechanism. Such interjections should be as brief as possible, to limit talking over one another. For example, "Point of Order - we are out of our allotted time today", might be a suitable interjection.
```

Comment:
This says that for urgent points of order they can be made verbally. Is the implication that non-urgent points of order (are their any?) should be raised through the chat instead?

More generally, do we want to add an expectation around use of chat during the meeting? In a lot of calls I'm on we use the chat basically to have side discussions, add clarifying points, or short comments that don't warrant raising a hand. So far we haven't done much of that in council meetings, and I kind of like that. I think it's easier for us all to follow the discussion if we don't have a side stream discussion going.

Reply:
Yea I think having side conversations in chat is not a good idea, but yes, I think we have been using the chat for non-urgent points of order and I think we should continue doing so.